amy/consents/admin.py

```diff
@@ -1,5 +1,34 @@
+import logging
+
+from consents.models import Consent
+import django_rq
+from autoemails.models import Trigger
 from consents.models import Term, TermOption
 from django.contrib import admin
+from autoemails.base_views import ActionManageMixin
+from autoemails.actions import NewConsentRequiredAction
+
+logger = logging.getLogger("amy.signals")  # TODO... why signals logger?
+scheduler = django_rq.get_scheduler("default")
+redis_connection = django_rq.get_connection("default")
+
+
+def send_consent_email(request, term: Term) -> None:
+    jobs, rqjobs = ActionManageMixin.add(
```

Comment:
Saw ActionManageMixin used like this in a different view. Not certain if this is the correct way however

Reply:
This is okay.